client/post-editor/editor-revisions-list/item.jsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { isObject } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import PostTime from 'reader/post-time';
+
+class EditorRevisionsListItem extends PureComponent {
+	selectRevision = () => {
+		this.props.selectRevision( this.props.revision.id );
+	}
+
+	render() {
+		return (
+			<button
+				className="editor-revisions-list__button"
+				onClick={ this.selectRevision }
+				type="button"
+			>
+				<span className="editor-revisions-list__date">
+					<PostTime date={ this.props.revision.date } />
+				</span>
+				&nbsp;
```

Comment:
Good spotting, but local testing doesn't show any centring issues. I'll leave as-is.